examples/graphene.jl

```diff
@@ -13,7 +13,7 @@ Ecut = 15
 lattice = [4.659533614391621 -2.3297668071958104 0.0;
            0.0 4.035274479829987 0.0;
            0.0 0.0 15.117809010356462]
-C = ElementPsp(:C, load_psp("hgh/pbe/c-q4"))
+C = ElementPsp(:C, psp=load_psp("hgh/pbe/c-q4"))
```

Comment:
use `;` instead of `,` here, it's more consistent for keyword arguments (it's not applied consistently in the code base but I try to apply it when I can)